_episodes_rmd/07-vector-shapefile-attributes-in-r.Rmd

```diff
@@ -328,7 +328,7 @@ Note that we could also use `aes(size = TYPE)` to tie the line thickness to the
 > >
 > > Now we can create our plot.
 > >
-> > ```{r harv-path-line-types}
+> > ```{r harv-path-line-types fig.cap="Roads and trails in the area with different line thickness for each type of paths."}
```

Comment:
```suggestion
> > ```{r harv-path-line-types, fig.cap="Roads and trails in the area with different line thickness for each type of paths."}
```